### ISSUE TITLE
Implement "real" unit tests

### ISSUE THREAD

**Issue by elenoir** [SimplyUb/MultiChainJavaAPI#13]
For the moment, the unit tests are under "syso" forms.

We must implement real unit tests based on test framework such as assertj.

**Comment by luigima** [comment]
Yes. Especially tests like this one: https://github.com/SimplyUb/MultiChainJavaAPI/blob/e04f0b08a72b0bbfa14054ab3c2a4b4ea6313f16/src/test/java/multichain/command/RAWTransactionCommandTest.java#L50

RawTransactions are not parsed correctly at the moment. Tests would expose this. I currently have no time for fixing this and did my own small modifications (that are not a good solution either). But yes, we need tests indeed. And multichain v2 compatibility.
